### PR TITLE
Fix for Supernova.

### PR DESCRIPTION
**Pull request by esellier**
Added context menu for testing message-top notifications.

### CODE REVIEW

**Comment by esellier** [comment]
#18 : fixed

File: notificationbar/schema.json [hunk 1]
```diff
@@ -154,7 +167,7 @@
         "async": true,
         "parameters": [
           {
-            "type": "integer",
+            "type": "string",
```
Comment (jobisoft) [inline notificationbar/schema.json]: Why did you change the type of the id?

File: notificationbar/schema.json [hunk 2]
```diff
@@ -96,13 +101,21 @@
                 "type": "string",
                 "pattern": "^(normal|bold(er)?|lighter|1000|([1-9]{1}[0-9]{0,2}))$"
               },
-              "^(background-)?color$": {
+              "^background-color$": {
```
Comment (jobisoft) [inline notificationbar/schema.json:120]: Could you give some details for the changes made to the pattern of the allowed css here and below?

**Review by jobisoft — COMMENTED** [review]
Thanks for your work on this, the addition to the test background is really helpful!

I simplified the PR and minimized the changeset. I also removed the unused `tabIndex` and `id` properties which were added by this PR, but are unused.

I have marked two places in the schema file. I do not know why you made the notificationId in the schema a string, and I also do not understand the changes to the css pattern. Could you give some details?

File: notificationbar/schema.json (hunk 1 above)
Comment (esellier) [reply notificationbar/schema.json]: Sorry... it's just that I use strings instead of integers in my Addon.

File: notificationbar/schema.json (hunk 2 above)
Comment (jobisoft) [reply notificationbar/schema.json:120]: I cloned the pattern from here to color, so i think we are good.

File: notificationbar/schema.json (hunk 1 above)
Comment (jobisoft) [reply notificationbar/schema.json]: Returning to integer for backward compatibility
